e2e/common.go

```diff
@@ -77,8 +78,10 @@ func SetExecuteBit(t *testing.T) {
 			return err
 		}
 
-		if err := os.Chmod(path, 0777); err != nil {
```

Comment:
It seems this was previously setting *everything* to be executable, not just shell scripts!

Reply:
Oops, good catch!